src/backend/cuda/texture.hpp

```diff
@@ -0,0 +1,66 @@
+/*******************************************************
```

Comment:
Can you rename this file to LookupTable1D.hpp?

Reply:
Okay